advisories/a84144.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Technical Advisory 84144
+advisory: A-84144
+summary: Multi-region tables whose locality has been altered to <code>REGIONAL BY ROW</code> are at risk of being corrupted
+toc: true
+affected_versions: v22.1.0-v22.1.3
+advisory_date: 2022-07-18
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+The problem affects multi-region tables whose locality has been altered to `REGIONAL BY ROW` in clusters running versions v22.1.0 to v22.1.3. The problem occurs when dropping a region: the drop may proceed when instead it should have yielded an error. Any subsequent attempt at changing the locality of the table will fail with an internal error.
```

Comment:
Suggest adding links to:

-  info re: `REGIONAL BY ROW` tables (eg https://www.cockroachlabs.com/docs/stable/multiregion-overview#regional-by-row-tables)
- v22.1 release notes page
- DROP REGION docs: https://www.cockroachlabs.com/docs/v21.1/drop-region.html